src/translations/cs.json

```
@@ -0,0 +1,41 @@
+{
+  "log_in": "Přihlásit se",
+  "log_out": "Odhlásit se",
+  "loged_in_as": "Přihlášen jako",
+  "logged_in": "Přihlášen",
+  "logging_in": "Přihlášení",
+  "logging_out": "Odhlášení",
+  "sign_up": "Zaregistrovat se",
+  "signing_up": "Registrace",
+  "remember_password": "Zapomněli jste heslo?",
```

Oh, forgot about this one, but similarly to the Slovak version – I noticed that the 'Forgot password?' link didn't get translated. I think it might be because the prop name for it is different?

```suggestion
  "forgot_password": "Zapomněli jste heslo?",
```